### DISCUSSION TITLE
Add a audio button

### DISCUSSION THREAD

**Discussion by Dixxi0509** [tahoma2d/tahoma2d#1245] [Ideas]
![download (1)](https://github.com/tahoma2d/tahoma2d/assets/91355354/863a5f51-8971-4a9d-8d2f-3d4d66578762)

In tahoma2d it let's you add audio but dragging and drop

But what if you can just import it with a button like csp(clip studio paint) has I think it'll be fun to have it (only if you want)

**Comment by artisteacher** [comment]
You can already import audio from the menu - with the load level command - this works for supported audio, supported video, single images, and image sequences.
